### ISSUE TITLE
[Bug] SendExchangeEmailWithAttachments: AttachmentFromString attachments are duplicated

### ISSUE THREAD

**Issue by HiQ-wipa** [CommunityHiQ/Frends.Community.Email#65]
# Description

If the task SendExchangeEmailWithAttachments is given multiple AttachmentFromString attachments, then the first attachment is duplicated.

# Steps to reproduce

Call SendExchangeEmailWithAttachments with 2 AttachmentFromString attachments.

# Expected behaviour

Each attachment should only appear once in the resulting email.

# Notes

This occurs because the [allAttachmentFilePaths loop](https://github.com/CommunityHiQ/Frends.Community.Email/blob/11cf42b6a70daf77c124070ad0bd4deeec40f30a/Frends.Community.Email/EmailTask.cs#L262) is contained within the [attachments loop](https://github.com/CommunityHiQ/Frends.Community.Email/blob/11cf42b6a70daf77c124070ad0bd4deeec40f30a/Frends.Community.Email/EmailTask.cs#L227). Abridged, the code looks like this:
```C#
foreach (var attachment in attachments) {
    allAttachmentFilePaths.Add(getFilePath(attachment));
    foreach (var filePath in allAttachmentFilePaths) {
        attachFileToEmail(filePath, email);
    }
}
```

In other words, with 2 attachments, the first iteration of the outer loop attaches the first attachment, and the second iteration attaches _the first attachment again_ and then the second attachment.
